src/ui/recording-preview.js

```diff
@@ -38,6 +38,11 @@ const RecordingPreview = ({ deviceType, title, type, url }) => {
           transform: 'translate(-50%, -50%)'
         }}
         src={url}
+
+        // Without this, some browsers show a black video element instead of the first frame.
```

Comment:
Just drop the comments.

Reply:
Why exactly? If I were to encounter the line of code I've added, I would be confused. The line is not really self-explaining and seems redundant, so I think a comment is useful here. Maybe I'm missing something...

Reply:
Did you find a bug report for this? Why not link to this? 

I should have added my comment two lines down after these 2 not very useful ones:
```
    // onLoadedMetadata={e => e.target.currentTime = 0}
     // onLoadedData={e => e.target.currentTime = 0}
```

BTW generally React does not like this style of comments. See here:

https://reactjs.org/docs/faq-build.html#how-can-i-write-comments-in-jsx

Reply:
Oh gosh how did I miss these two useless lines? Sorry, of course, those comments are useless. Thanks for catching!

> Did you find a bug report for this?

Unfortunately, I did not.

Reply:
> BTW generally React does not like this style of comments. See here:

I found that, too, but interestingly, this leads to parsing errors for me. Not sure why.

```
./src/ui/recording-preview.js
  Line 42:98:  Parsing error: Unexpected token, expected "..."

  40 |         src={url}
  41 | 
> 42 |         {/* Without this, some browsers show a black video element instead of the first frame. */}
     |                                                                                                  ^
```

Reply:
Comments must be outside of tags. That's all :-)

Reply:
Oh ok. Would you prefer me to change it?